graphstorm-processing/graphstorm_processing/data_transformations/dist_transformations/dist_hf_transformation.py

```diff
@@ -112,6 +112,7 @@ def tokenize(text):
         logging.warning("The device to run huggingface transformation is %s", device)
         tokenizer = AutoTokenizer.from_pretrained(hf_model)
         if max_seq_length > tokenizer.model_max_length:
+            # TODO: Could we possibly raise this at config time?
```

Comment:
We could try to instantiate a Tokenizer at config time, to see if the seq length is valid?

Reply:
Usually when we are trying to instantiate a Tokenizer on each executor, it will take some time and effort. I do not recommend to do so. As we are going to do it multiple times if adding a separate check.